src/Apps/ArtAdvisor/06-Discovery-Neartext/server.ts

```diff
@@ -0,0 +1,101 @@
+import express from "express"
+import { ArtsyRequest, ArtsyResponse } from "Server/middleware/artsyExpress"
+import _ from "lodash"
+import { WeaviateDB } from "./weaviate-db"
+import { generateUuid5 } from "weaviate-ts-client"
+
+const weaviateDB = new WeaviateDB()
```

Comment:
The Weaviate wrapper gets instantiated once here, and is used throughout the server endpoint code as our abstraction over the vector database.